HLTrigger/Muon/plugins/HLTTriMuonIsolation.h

```diff
@@ -64,7 +64,7 @@ class HLTTriMuonIsolation : public edm::global::EDProducer<> {
   const bool EnableAbsIso_;
 };
 
-HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
+inline HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
```

Comment:
do we need the `inline` also in the method declaration inside the class body ?

Reply:
there is no need for that.